client/state/data-layer/wpcom/posts/revisions/index.js

```diff
@@ -64,8 +74,17 @@ export const receiveError = ( { dispatch }, { siteId, postId }, next, rawError )
  * @param {Array} revisions raw data from post revisions API
  */
 export const receiveSuccess = ( { dispatch }, { siteId, postId }, next, revisions ) => {
+	const normalizedRevisions = map( revisions, normalizeRevision );
+
+	forEach( normalizedRevisions, ( revision, index ) => {
+		revision.changes = index === normalizedRevisions.length - 1
+			? { added: 0, removed: 0 }
+			: countDiffWords( diffWords( normalizedRevisions[ index + 1 ].content, revision.content ) );
```

Comment:
Curious what's the reasoning behind computing the diff when fetching the revisions? Naively, I'd have let this for a memoized selector, for example, assuming this can be heavy and compute the diff only when needed.

Maybe not needed but what if we want to diff an old revision with the last one for example.